.circleci/windows/erlang_install.bat

```diff
@@ -0,0 +1,20 @@
+@echo off
+IF NOT "x%1"=="x" IF NOT "x%2"=="x" GOTO:INSTALL
+@echo Usage: %0 ^<otp_version^> ^<install_path^>
+exit /b 2
+
+:INSTALL
+@echo %time% : Install Erlang/OTP %1
+IF EXIST %2 GOTO:INSTALLED
+
+SET "URL=https://packages.erlang-solutions.com/erlang/erlang/esl-erlang/FLAVOUR_1_general/esl-erlang_%1~windows_amd64.exe"
```

Comment:
This is a much faster download mirror, not sure if reliable. Official site download takes some minutes vs few seconds here